static/src/stylesheets/module/facia/item-layouts/_fc-item--three-quarters.scss

```diff
@@ -65,6 +65,7 @@ Three quarter item. Looks like a wide standard, a bit like this:
         .fc-item__content {
             width: auto;
             max-width: gs-span(5);
+            @include flex-basis(initial);
```

Comment:
should be in a `.has-flex-wrap`, does it also work on non-flex? could do with some browserstack…